docker-compose.yaml

```diff
@@ -63,7 +63,7 @@ x-airflow-common:
     AIRFLOW_VAR_MARC_S3_BUCKET: 'sinopia-marc-development'
     AIRFLOW_VAR_RDF2MARC_LAMBDA: 'sinopia-rdf2marc-development'
     AIRFLOW_VAR_SINOPIA_ENV: ${AIRFLOW_VAR_SINOPIA_ENV:-dev}
-    AIRFLOW_VAR_SQS_URL: 'http://localstack:4566/000000000000/'
+    #AIRFLOW_VAR_SQS_URL: 'http://localstack:4566/000000000000/'
```

Comment:
Have you tried testing with this variable with the update to Airflow?

Reply:
working again with the airflow update